### DISCUSSION TITLE
I cannot upgrade my Student Pack

### DISCUSSION THREAD

**Discussion by marvinhuelsmann** [community/community#66058] [GitHub Education]
### Hi everyone,

I already have the GitHub Student Pack and would like to extend it for a year but always come:

**You were last verified as a student on Oct 22, 2022. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.**

**Comment by Gatix1** [comment]
I think it says that you don't need to reverify it yet, and you can't do it now. When the period will end you will be able to reverify i guess
